cumulusci.yml

```diff
@@ -645,6 +645,34 @@ flows:
             0.1:
                 flow: translations
 
+    customer_org:
+        steps:
+            1:
+                flow: dependencies
+                ui_options:
+                    deploy_pre:
+                        acc_record_types:
+                            name: EDA - Account Record Types
+                        contact_key_affl_fields:
+                            name: EDA - Contact Key Affiliation Fields
+            2:
+                task: install_managed
+            3:
+                task: deploy_post
+                ui_options:
+                    course_connection_record_types:
+                        name: EDA - Course Connection Record Types
+                    facility_display_name:
+                        name: EDA - Facility Display Name Formula Field
+                    case_behavior_record_types:
+                        name: EDA - Case Behavior Record Types
+                    health_check_tab:
+                        name: Health Check
```

Comment:
Can we move these to a task override to make them easier to consume downstream? I'd been meaning to do so during the SAL refactor, but lost it in the shuffle.

Reply:
For example, we're doing this in SAL:
```yaml
    # Shared UI Options
    deploy_pre:
        ui_options:
            src:
                name: Advisor Link - Prerequisite Record Types
    deploy_post:
        ui_options:
            src:
                name: Advisor Link - Unpackaged Metadata
```

Reply:
Oh, good call. Done.